Indexing/TermIndex.cpp

```diff
@@ -427,33 +457,33 @@ void HeuristicInstantiationIndex::handleClause(Clause* c, bool adding)
         /*cout << "lhs is " + lit->nthArgument(0)->toString() << endl;
         cout << "arg " + leftArgs[i].toString() << endl;*/
         cout << "inserting " + lambdaTerm->toString() << endl;
-        cout << "inserting " + combTerm.toString() << endl;          
+        cout << "inserting " + combTerm.toString() << endl;
```

Comment:
leftover debug printing?

Reply:
But this isn't your code... I don't think it should be here though.

Reply:
Should I remove it completely or just comment it out? (There's also similar debug printing in lines 475-478.)

Reply:
I think the least disruptive thing is to comment these print statements out. Thanks for dealing with this here - it must have been missed in a previous PR.

Reply:
Done.